packages/vouching/contracts/ZepToken.sol

```diff
@@ -1,29 +1,33 @@
 pragma solidity ^0.4.24;
 
-import 'openzeppelin-solidity/contracts/token/ERC20/PausableToken.sol';
-import 'openzeppelin-solidity/contracts/token/ERC20/StandardToken.sol';
+import './TPL/TPLToken.sol';
 
 /**
  * @title ZepToken
  * @dev ZEP token contract including mintable, pausable and burnable functionalities
  */
-contract ZepToken is StandardToken, PausableToken {
+contract ZepToken is TPLToken {
 
-  string public constant name = "Zep Token";
-  string public constant symbol = "ZEP";
-  uint8 public constant decimals = 18;
+  function initialize(
+    AttributeRegistry _jurisdictionAddress,
+    uint256 _validRecipientAttributeId
+  ) 
+    isInitializer("ZEPToken", "1.0.0")
+    public
+  {
+    uint8 decimals = 18;
+    uint256 TOTAL_SUPPLY = 1e8 * (10 ** uint256(decimals));
+    TPLToken.initialize(
+      msg.sender, 
+      "Zep Token", 
+      "ZEP",
+      decimals,
+      TOTAL_SUPPLY,
+      _jurisdictionAddress,
+      _validRecipientAttributeId
+    );
 
-  uint256 public constant TOTAL_SUPPLY = 1e8 * (10 ** uint256(decimals));
-
-  /**
-   * @dev Constructor that gives msg.sender all of existing tokens, and sets it as the owner
-   */
-  constructor() public {
-    owner = msg.sender;
-    totalSupply_ = TOTAL_SUPPLY;
-    balances[msg.sender] = TOTAL_SUPPLY;
     emit Transfer(address(0), msg.sender, TOTAL_SUPPLY);
```

Comment:
Remove this event, as the base contract already fires it (see [here](https://github.com/OpenZeppelin/openzeppelin-zos/blob/master/contracts/token/ERC20/DetailedPremintedToken.sol#L27))